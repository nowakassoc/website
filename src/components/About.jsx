import React from 'react';

const About = () => (
  <section id="section-about">
    <span id="about" className="section-link" />
    <h2>About Martin Nowak</h2>
    <p>
      Martin C. Nowak has more than thirty years experience in healthcare;
      twenty three of those years at the executive level. From 2002 to 2005 he
      was Chief Planning and Strategy Officer for the UAB Health System; from
      1998 to 2002 he was interim then appointed Chief Executive Officer of UAB
      Hospital; and, from 1981 to 1998 he served in various executive hospital
      positions including Chief Executive Officer of St. Clair Regional Hospital
      and Administrator of the UAB Psychiatric Hospital. From 1992 to 1994 he
      served as the Associate Executive Director for Information Systems (Chief
      Information Officer) at UAB Hospital. With recognized success, Mr. Nowak
      has directed hospitals from 90 to 908 beds; budgets exceeding $1.4 billion
      revenue; and, workforce exceeding 5,000 employees. Mr. Nowak has
      considerable consulting experience including distressed hospital
      turnarounds.
    </p>
    <p>
      Mr. Nowak is cofounder and CEO of{' '}
      <a
        aria-label="ViM Analytics, LLC"
        href="http://www.vimanalytics.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        ViM Analytics, LLC
      </a>
      , a market research firm specializing in collecting and quantifying
      hospital visitor experiences. Hospital visitors, which include caregivers,
      family, friends, and even clergy members, are the most ignored demographic
      in healthcare today but are eager participants in the caregiving process.
      ViM&apos;s proprietary process utilizes the opinions and insights of
      visitors to drive actionable change in healthcare settings. ViM findings
      can be used to drive positive visitor experiences, which result in higher
      satisfaction scores, improved community engagement, and ultimately,
      improvements in volume and revenue.
    </p>
    <p>
      He has served on numerous boards including the Alabama Hospital
      Association from 2001 to 2005. The recipient of numerous awards, Mr. Nowak
      was named a &ldquo;Top 100 Who&apos;s Who in Healthcare&rdquo; by the
      Birmingham Business Journal (2002 and 2007) and received the
      &ldquo;Distinguished Service Award&rdquo; from the Alabama Hospital
      Association (2002). Mr. Nowak is a frequent public speaker, and is often
      sought for opinion by news media. He has been cited and interviewed on
      many occasions by healthcare trade journals. From 2003 - 2011 he served
      via gubernatorial appointment on the Alabama State Health Coordinating
      Council. He co-authored the health care strategic plan for indigent
      residents of Jefferson County, Alabama. Mr. Nowak is a Mentor in the UAB
      Collat School of Business Innovation Depot.
    </p>
    <p>
      In 2005, he was named the first Outstanding Alumnus of UAB&apos;s School
      of Natural Science and Mathematics. In 2014 he was named Alumnus of the
      Year by the UAB Graduate School of Health Administration. And, in 2018 he
      was named UAB Distinguished Biology Alumnus.
    </p>
    <p>
      Mr. Nowak has been a principle in several startup companies and is locally
      recognized for his entrepreneurial spirit. He holds several patents
      including two medical claims data mining processes to illuminate quality
      and other issues in care delivery. He has licensed another commercial
      product patent to a national firm. Mr. Nowak earned both B.S. and M.H.A.
      degrees from the University of Alabama at Birmingham.
    </p>
  </section>
);

export default About;
