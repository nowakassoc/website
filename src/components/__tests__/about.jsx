import React from 'react';
import { render } from 'react-testing-library';
import About from '../About';

it('About renders correctly', () => {
  const tree = render(<About />);
  expect(tree).toMatchSnapshot();
});
