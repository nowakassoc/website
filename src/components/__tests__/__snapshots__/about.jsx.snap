// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        id="section-about"
      >
        <span
          class="section-link"
          id="about"
        />
        <h2>
          About Martin Nowak
        </h2>
        <p>
          Martin C. Nowak has more than thirty years experience in healthcare; twenty three of those years at the executive level. From 2002 to 2005 he was Chief Planning and Strategy Officer for the UAB Health System; from 1998 to 2002 he was interim then appointed Chief Executive Officer of UAB Hospital; and, from 1981 to 1998 he served in various executive hospital positions including Chief Executive Officer of St. Clair Regional Hospital and Administrator of the UAB Psychiatric Hospital. From 1992 to 1994 he served as the Associate Executive Director for Information Systems (Chief Information Officer) at UAB Hospital. With recognized success, Mr. Nowak has directed hospitals from 90 to 908 beds; budgets exceeding $1.4 billion revenue; and, workforce exceeding 5,000 employees. Mr. Nowak has considerable consulting experience including distressed hospital turnarounds.
        </p>
        <p>
          Mr. Nowak is cofounder and CEO of
           
          <a
            href="http://www.vimanalytics.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            ViM Analytics, LLC
          </a>
          , a market research firm specializing in collecting and quantifying hospital visitor experiences. Hospital visitors, which include caregivers, family, friends, and even clergy members, are the most ignored demographic in healthcare today but are eager participants in the caregiving process. ViM's proprietary process utilizes the opinions and insights of visitors to drive actionable change in healthcare settings. ViM findings can be used to drive positive visitor experiences, which result in higher satisfaction scores, improved community engagement, and ultimately, improvements in volume and revenue.
        </p>
        <p>
          He has served on numerous boards including the Alabama Hospital Association from 2001 to 2005. The recipient of numerous awards, Mr. Nowak was named a “Top 100 Who's Who in Healthcare” by the Birmingham Business Journal (2002 and 2007) and received the “Distinguished Service Award” from the Alabama Hospital Association (2002). Mr. Nowak is a frequent public speaker, and is often sought for opinion by news media. He has been cited and interviewed on many occasions by healthcare trade journals. From 2003 - 2011 he served via gubernatorial appointment on the Alabama State Health Coordinating Council. He co-authored the health care strategic plan for indigent residents of Jefferson County, Alabama. Mr. Nowak is a Mentor in the UAB Collat School of Business Innovation Depot.
        </p>
        <p>
          In 2005, he was named the first Outstanding Alumnus of UAB's School of Natural Science and Mathematics. In 2014 he was named Alumnus of the Year by the UAB Graduate School of Health Administration. And, in 2018 he was named UAB Distinguished Biology Alumnus.
        </p>
        <p>
          Mr. Nowak has been a principle in several startup companies and is locally recognized for his entrepreneurial spirit. He holds several patents including two medical claims data mining processes to illuminate quality and other issues in care delivery. He has licensed another commercial product patent to a national firm. Mr. Nowak earned both B.S. and M.H.A. degrees from the University of Alabama at Birmingham.
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      id="section-about"
    >
      <span
        class="section-link"
        id="about"
      />
      <h2>
        About Martin Nowak
      </h2>
      <p>
        Martin C. Nowak has more than thirty years experience in healthcare; twenty three of those years at the executive level. From 2002 to 2005 he was Chief Planning and Strategy Officer for the UAB Health System; from 1998 to 2002 he was interim then appointed Chief Executive Officer of UAB Hospital; and, from 1981 to 1998 he served in various executive hospital positions including Chief Executive Officer of St. Clair Regional Hospital and Administrator of the UAB Psychiatric Hospital. From 1992 to 1994 he served as the Associate Executive Director for Information Systems (Chief Information Officer) at UAB Hospital. With recognized success, Mr. Nowak has directed hospitals from 90 to 908 beds; budgets exceeding $1.4 billion revenue; and, workforce exceeding 5,000 employees. Mr. Nowak has considerable consulting experience including distressed hospital turnarounds.
      </p>
      <p>
        Mr. Nowak is cofounder and CEO of
         
        <a
          href="http://www.vimanalytics.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          ViM Analytics, LLC
        </a>
        , a market research firm specializing in collecting and quantifying hospital visitor experiences. Hospital visitors, which include caregivers, family, friends, and even clergy members, are the most ignored demographic in healthcare today but are eager participants in the caregiving process. ViM's proprietary process utilizes the opinions and insights of visitors to drive actionable change in healthcare settings. ViM findings can be used to drive positive visitor experiences, which result in higher satisfaction scores, improved community engagement, and ultimately, improvements in volume and revenue.
      </p>
      <p>
        He has served on numerous boards including the Alabama Hospital Association from 2001 to 2005. The recipient of numerous awards, Mr. Nowak was named a “Top 100 Who's Who in Healthcare” by the Birmingham Business Journal (2002 and 2007) and received the “Distinguished Service Award” from the Alabama Hospital Association (2002). Mr. Nowak is a frequent public speaker, and is often sought for opinion by news media. He has been cited and interviewed on many occasions by healthcare trade journals. From 2003 - 2011 he served via gubernatorial appointment on the Alabama State Health Coordinating Council. He co-authored the health care strategic plan for indigent residents of Jefferson County, Alabama. Mr. Nowak is a Mentor in the UAB Collat School of Business Innovation Depot.
      </p>
      <p>
        In 2005, he was named the first Outstanding Alumnus of UAB's School of Natural Science and Mathematics. In 2014 he was named Alumnus of the Year by the UAB Graduate School of Health Administration. And, in 2018 he was named UAB Distinguished Biology Alumnus.
      </p>
      <p>
        Mr. Nowak has been a principle in several startup companies and is locally recognized for his entrepreneurial spirit. He holds several patents including two medical claims data mining processes to illuminate quality and other issues in care delivery. He has licensed another commercial product patent to a national firm. Mr. Nowak earned both B.S. and M.H.A. degrees from the University of Alabama at Birmingham.
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
